assets/dev/js/editor/controls/color.js

```diff
@@ -66,57 +46,36 @@ export default class extends ControlBaseDataView {
 		this.$pickerButton = jQuery( this.colorPicker.picker.getRoot().button );
 
 		this.$pickerButton.tipsy( {
-			title: () => this.getControlValue() ? this.getColorObject().displayCode : '',
+			title: () => this.getControlValue() ? this.getNewGlobalData().code : '',
 			offset: 4,
 			gravity: () => 's',
 		} );
 	}
 
-	getColorObject() {
-		const color = this.colorPicker.picker.getColor(),
-			colorObject = {};
-
-		colorObject.displayCode = color.toHEXA().toString( 0 );
-		// color.a is the transparency percentage. 1 means HEX value and not rgba()
-		if ( 1 === color.a ) {
-			colorObject.code = colorObject.displayCode;
-		} else {
-			colorObject.code = color.toRGBA().toString( 0 );
-		}
-
-		colorObject.name = this.getColorName( colorObject );
-
-		return colorObject;
+	getNewGlobalData() {
```

Comment:
Think about a better name for this method

Reply:
Done `getGlobalData`